README.md

```diff
@@ -127,3 +127,85 @@ To access the WebCSD service locally go to http://localhost in a browser.
 ## Contact support
 
 If you experience any difficulties with installing or using On-Site WebCSD, please contact our support team at support@ccdc.cam.ac.uk who will be happy to assist you.
+
+## Kubernetes support
```

Comment:
Is this more suitable for the wiki?

Reply:
I did wonder...possibly one for the Support section?

Reply:
I was looking at the wiki and it might be best under "Alternative ways to setup On Site WebCSD" ?